pkg/operator/helpers/helpers.go

```
@@ -36,6 +36,8 @@ import (
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/component-base/featuregate"
+
```

golangci-lint (2.12.1)

[error] File is not properly formatted

(gci)